From a4c292229c7c2cfb49f88c4741f0c2d25358d120 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Fri, 6 Mar 2026 16:02:17 +0100
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 1e6b7a21..053ba9f5 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,7 +24,7 @@ dependencies = [
     "numpy>=2.0",
     "h5py>=3.9.0",
     "pandas>=2.0",
-    "nglview>=3.0.5",
+    "nglview>=0.0.0",
     "ase>=3.23",
     "plotly>=5.23",
     "kaleido>=1.0",
